share/wake/lib/core/list.wake

```diff
@@ -67,14 +67,26 @@ export def tail: List a => List a = match _
     Nil  = Nil
     _, t = t
 
-# map: create a new List by applying a function f to each element of a List.
-#
-#   map str    (seq 5) = "0", "1", "2", "3", "4", Nil
-#   map (_+10) (seq 5) = 10, 11, 12, 13, 14, Nil
-export def map (f: a => b): List a => List b =
-    def loop = match _
+# Create a new List by applying the function *mapFn* to each element of *list*.
```

Comment:
Any reason you chose to use italics (`*`) instead of monospace (<code>`</code>) for variable/parameter names? I think monospace formatting is more common in documentation comments across various languages.

Reply:
So I tried using `.

It looks ok in vscode:
![Screen Shot 2021-09-14 at 6 09 32 PM](https://user-images.githubusercontent.com/1101706/133354006-e99f5987-0ce9-47a8-bf1f-288022ee6e9d.png)

But see below for neovim, where it does not work at all.

Reply:
I think stylistically and for neovim support, I prefer using *. However, I agree that most people would naturally write a `. It's also probably fine if someone does that, because they have not read the style guide. However, since the style guide defines what people SHOULD do, I think we're not really bound, either way.

What do you guys think?

Reply:
Ok. Backticks.